src/components/Projects.js

```diff
@@ -0,0 +1,165 @@
+/* eslint-disable react/no-array-index-key */
+import React from 'react';
+import styled from 'styled-components';
+import FeaturedProject from '../FeaturedProject.json';
+
+export const Projects = () => {
+  const GITHUB_NAME = 'LisaVasic';
+
+  return (
+    <ProjectOuterWrapper>
+      <ProjectInnerWrapper>
+        <h5>FEATURED PROJECTS</h5>
+        <FeaturedProjectWrapper>
+          {FeaturedProject.map((project) => {
+            return (
+              <ProjectCard
+                key={project.title}
+                href={`https://github.com/${GITHUB_NAME}/${project['repo-name']}`}
+                target="_blank">
+                <ThumbnailWrapper url={project.image}>
+                  <ThumbnailTitle>{project.title.toUpperCase()}</ThumbnailTitle>
+                </ThumbnailWrapper>
+                <ProjectInfoHeader>
+                  {project['title-description'].toUpperCase()}
+                </ProjectInfoHeader>
+                <ProjectInfo>{project['project-description']}</ProjectInfo>
+                <ul style={{ padding: 0 }}>
+                  {project.tags.map((tag) => (
+                    <ProjectTags>
+                      {tag}
+                    </ProjectTags>
+                  ))}
+                </ul>
+              </ProjectCard>
+            );
+          })}
+        </FeaturedProjectWrapper>
```

Comment:
Wonderful work with the first featured project, Movies! Though, when clicking on the other 3 projects I'm ending up on wrong pages. Looks like some changes needs to be done in the json :)